### PR TITLE
Refactor code, primarily by moving everything under src

### PR DESCRIPTION
**Pull request by young-k**
Architectural change, some minor changes within files as well. Verified that all functionality is preserved.

### CODE REVIEW

**Review by meganle — APPROVED** [review]
LGTM! Just remove unused imports

File: app.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import data
 from flask import Flask
 from flask_graphql import GraphQLView
 from graphene import Schema
-from schema import Query
 
-app = Flask(__name__)
+from src import data
```
[the remark this answers is not in the document]
Comment (kq-li) [reply app.py:5]: go get em hound

**Comment by young-k** [comment]
those were already unused i just refactored the code 😠, but will do

File: src/data.py [hunk 2]
```diff
@@ -69,7 +70,7 @@ def parse(data_json):
     eateries[new_eatery.id] = new_eatery
 
 def get_image_url(slug):
-  return IMAGES_URL + slug + '.jpg'
+  return "{}{}.jpg".format(IMAGES_URL, slug)
```
Comment (kq-li) [inline src/data.py:73]: single quotes you goof

**Review by kq-li — APPROVED** [review]
good changes A+ work

**Pull request merged**